signing_clients/apps.py

```diff
@@ -102,6 +106,21 @@ def _digest(data):
     return {'md5': md5.digest(), 'sha1': sha1.digest()}
 
 
+def get_addon_type_from_signature(pkcs7):
+    info = SignatureInfo(pkcs7)
+    ou = get_signer_organizational_unit_name(info)
+
+    if ou == constants.OU_PRODUCTION:
+        # TODO: where should the CN be taken from to identify
+        # hotfix-addons? `if endEntity.CN = "firefox-hotfix@mozilla.org"`
```

Comment:
Perfect! So it's exactly https://github.com/mozilla/signing-clients/pull/28/files#diff-0eaa4a7d5a7900fca93854da27046e92R414 just the correct wording. Thanks!